sdks/python/apache_beam/io/gcp/gcsio.py

```diff
@@ -483,6 +489,7 @@ def size(self):
     return self._size
 
   def get_range(self, start, end):
+    self._download_stream.seek(0)
```

Comment:
Why is this needed in Python 3?

Reply:
Difference in behavior between cStringIO and BytesIO. In BytesIO truncate doesn't move the file pointer, see [here](https://stackoverflow.com/questions/39109068/why-does-truncating-a-bytesio-mess-it-up)

Reply:
Yes, thanks for the clarification.